exporter/sumologicexporter/config.go

```diff
@@ -133,3 +133,7 @@ const (
 	// DefaultGraphiteTemplate defines default template for Graphite
 	DefaultGraphiteTemplate string = "%{_metric_}"
 )
+
+func (cfg *Config) Validate() error {
```

Comment:
[exporter/sumologic] Ensure Validate implementation does the right thing

Reply:
OK, I think that works for start and then some part of the "initExporter" will need to be extracted as the validation function. I am going to create a separate PR (I guess after this one is merged?)